atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/transaction/impl/SerializableTransactionManagerTest.java

```diff
@@ -46,76 +48,100 @@
     private TimelockService mockTimelockService = mock(TimelockService.class);
     private Cleaner mockCleaner = mock(Cleaner.class);
     private AsyncInitializer mockInitializer = mock(AsyncInitializer.class);
-    private Runnable mockCallback = mock(Runnable.class);
+    private Callback mockCallback = mock(Callback.class);
 
     private SerializableTransactionManager manager;
 
     @Before
     public void setUp() {
-        when(mockKvs.isInitialized()).thenReturn(false);
+        setInitializationStatus(false, false, false, false);
```

Comment:
optional suggestion: since we're extracting helper methods... it's mostly "false * 4" or "true * 4", so consider adding e.g. `whenEverythingIsInitialized` and `whenNothingIsInitialized`.